cms/static/js/views/settings/main.js

```diff
@@ -164,7 +209,26 @@ var DetailsView = ValidatingView.extend({
         $(e.currentTarget).attr('title', currentTimeText);
     },
     updateModel: function(event) {
+        var value;
+        var index = event.currentTarget.getAttribute('data-index');
         switch (event.currentTarget.id) {
+        case 'course-learning-info-' + index:
```

Comment:
`learning_info` and `instructor_info` are collections on the model, updating the structure of those collections (adding/removing items) is handled elsewhere, but changes to the individual values/object fields are handled here.